assets/search/search.controller.js

```diff
@@ -136,18 +141,18 @@ module.exports = function ($scope, $timeout, UserFactory) {
       url += "&user=" + vm.input.user;
     }
     url += "&skip=" + skip;
-
-    var searchedFor = url;
+    vm.done = false;
     io.socket.get(url, function (data, res) {
-      if (res.statusCode === 200 && searchedFor === url) {
+      if (res.statusCode === 200) {
```

Comment:
This needs to be here, because otherwise we will put in the results for an old search.

Example:

* User types "YaMan" and stops over the timeout time
* System requests a search for that search
* User deletes "YaMan" and starts typing "Darkr" before results come back
* Results come back, and user is seeing results for "YaMan" when they are searching for "Darkr"